web/src/components/tasks/anytime-action/AnytimeActionSwitchComponent.tsx

```diff
@@ -9,6 +10,13 @@ interface Props {
 }
 
 export const AnytimeActionSwitchComponent = (props: Props): ReactElement => {
+  // TODO: when the feature flag is removed, the tax clearance filename can be added to the switch statement
```

Comment:
Create ticket in ADO to remove feature flag and content associated with it.

Reply:
@cbrunefearless @FarazA22 can we make this ticket and link it here before we merge the PR?

Reply:
will make a ticket for this Friday 3/7 and link it here

Reply:
https://www.pivotaltracker.com/story/show/188963481 / https://dev.azure.com/NJInnovation/Business%20First%20Stop/_workitems/edit/13223